crates/gpio/src/backend.rs

```diff
@@ -173,7 +173,7 @@ fn start_backend<D: 'static + GpioDevice + Send + Sync>(args: GpioArgs) -> Resul
                     info!("Stopping cleanly.");
                 }
                 Err(vhost_user_backend::Error::HandleRequest(
-                    vhost_user::Error::PartialMessage,
+                    vhost_user::Error::PartialMessage | vhost_user::Error::Disconnected,
```

Comment:
What if we instead make this case fall as in the case of `Ok(())`?

Reply:
I am fine with that too. Just that it won't show up anything on the terminal running the daemon, and I am not sure if we should print something there or not :)

Reply:
Ehm, right, maybe could be useful to know that the VM is disconnected. I don't have a strong opinion too.